tests/functional/pod_and_daemons/test_csi_logs_rotation.py

```diff
@@ -0,0 +1,96 @@
+import time
+import logging
+
+from ocs_ci.framework.testlib import BaseTest
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.constants import OPENSHIFT_STORAGE_NAMESPACE
+from ocs_ci.framework.pytest_customization.marks import brown_squad, tier1
+
+log = logging.getLogger(__name__)
+LOGS_DIR_NAME = "/var/lib/rook/openshift-storage.cephfs.csi.ceph.com/log/node-plugin/"
+CURRENT_LOG_NAME = "csi-cephfsplugin.log"
+WAIT_FOR_ROTATION_TIME = 1200  # seconds
+
+
+@brown_squad
+@tier1
+class TestPodsCsiLogRotation(BaseTest):
+    def get_logs_details_on_pod(self, pod_obj):
+        """
+        Gets csi pod log files details
+
+        Args:
+            pod_obj (obj): Pod which log files should be investigated
+
+        Returns:
+            gz_logs_num (int): Number of compressed log files
+            current_log_file_size (int) Size of the current log file
+        """
+
+        all_logs = (
+            pod_obj.exec_cmd_on_pod(
+                command=f"-- ls -l {LOGS_DIR_NAME}",
+                container_name="log-collector",
+                out_yaml_format=False,
+                shell=True,
+            )
+            .strip()
+            .split("\n")
+        )
+        gz_logs_num = 0
+        current_log_file_size = 0
```

Comment:
Good to move this function to common libraries.

Reply:
Done, moved to pod.py.